shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -77,6 +78,38 @@ void Reset() {
 
 #pragma mark - Private interface declaration.
 
+/**
+ * Private interface declaration for FlutterWindowDelegate.
+ *
+ * This delegate inserts the FlutterTextInputPlugin into FlutterTextFields
+ * as their field editors.
+ *
+ * The FlutterTextField are only created when accessibility services are
+ * turned on. Therefore, this delegate does nothing if none of accessibility
+ * services is on.
+ */
+@interface FlutterWindowDelegate : NSObject <NSWindowDelegate>
+
+@end
+
+/**
+ * Private interface declaration for FlutterNativeTextFieldHidder.
+ *
+ * This NSView is the content view of the FlutterViewController. This view
+ * is the wrapper of the FlutterView, and it is useful to attach other native
+ * views, e.g. the FlutterTextField, without affecting the accessibility
+ * subtree of FlutterView.
```

Comment:
You've described this class very generically here, which I agree with, but then named the class as if the example use case is the only possible use it will ever have. The name should be something more generic that better matches this description, like "FlutterViewWrapper"